portfolio/src/main/webapp/script.js

```diff
@@ -11,12 +11,14 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-$(document).ready(function () {
-  $("a").click(function () {
+$(document).ready(function() {
+  getComments();
+
+  $("a").click(function() {
```

Comment:
Can you update prettier to use single quotes for JavaScript?

Reply:
can do and done!